tests/test_items.py

```diff
@@ -1,3 +1,5 @@
+from pprint import pprint
```

Comment:
Do you use `pprint` in some tests?

Reply:
Opss, you catch me! Fixed here: https://github.com/scrapinghub/autoextract-poet/pull/13/commits/de1b36cc2d9302da876381611415de9b630112e3